github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -1094,6 +1094,97 @@ periodics:
           hostPath:
             path: /dev/vfio/
             type: Directory
+- name: periodic-kubevirt-e2e-k8s-1.19-sriov-migration
+  annotations:
+    testgrid-dashboards: kubevirt-periodics
+    testgrid-days-of-results: "60"
+    k8s.v1.cni.cncf.io/networks: 'multus-cni-ns/sriov-passthrough-cni,multus-cni-ns/sriov-passthrough-cni'
+  cron: "0,30 2,3 * * *"
+  decorate: true
+  decoration_config:
+    timeout: 4h
+    grace_period: 30m
+  labels:
+    preset-dind-enabled: "true"
+    preset-docker-mirror-proxy: "true"
+    preset-shared-images: "true"
+  extra_refs:
+    - org: kubevirt
+      repo: kubevirt
+      base_ref: main
+      work_dir: true
+  skip_report: true
+  cluster: prow-workloads
+  reporter_config:
+    slack:
+      job_states_to_report: []
+  spec:
+    nodeSelector:
+      hardwareSupport: sriov-nic
+    affinity:
+      podAntiAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchExpressions:
+                - key: sriov-pod
+                  operator: In
+                  values:
+                    - "true"
+            topologyKey: kubernetes.io/hostname
+          - labelSelector:
+              matchExpressions:
+                - key: sriov-pod-multi
+                  operator: In
+                  values:
+                    - "true"
+            topologyKey: kubernetes.io/hostname
+    containers:
+      - image: quay.io/kubevirtci/golang:v20210316-d295087
+        env:
+          - name: KUBEVIRT_QUARANTINE
+            value: "true"
```

Comment:
Do we need to execute quarantined tests in this periodic?

Reply:
Sorry this should not be there I will remove it